### PR TITLE
cve-2010-3415 and cve-2017-5038

### PR DESCRIPTION
**Pull request by VigorousJunk**
Adding information about two chromium vulnerabilities to the cve files.

### CODE REVIEW

File: cves/CVE-2017-5038.yml [hunk 1]
```diff
@@ -98,20 +110,24 @@ discovered:
...
     What subsystems was the mistake in?
 
     Look at the path of the source code files code that were fixed to get
     directory names. Look at comments in the code. Look at the bug reports how
     the bug report was tagged. Examples: "clipboard", "gpu", "ssl", "speech", "renderer"
-  answer: 
-  name: 
+  answer: |
+    Guestview Renderer.
+  name:
```
Comment (axt1384) [inline cves/CVE-2017-5038.yml]: I'd just repeat the answer in the name section if you don't know the name of the subsystem

File: cves/CVE-2010-3415.yml [hunk 2]
```diff
@@ -49,31 +47,31 @@ bounty:
...
+  :note: Simplify Geolocation error message protocol
 upvotes_instructions: |
   For the first round, ignore this upvotes number.
 
   For the second round of reviewing, you will be giving a certain amount of
   upvotes to each vulnerability you see. Your peers will tell you how
   interesting they think this vulnerability is, and you'll add that to the
   upvotes score on your branch.
-upvotes: 
+upvotes:
```
Comment (axt1384) [inline cves/CVE-2010-3415.yml]: I'd give this CVE a 4

File: cves/CVE-2017-5038.yml [hunk 3]
```diff
@@ -32,42 +32,52 @@ description_instructions: |
...
+    Refactoring common functionality to reduce lines.
 upvotes_instructions: |
   For the first round, ignore this upvotes number.
 
   For the second round of reviewing, you will be giving a certain amount of
   upvotes to each vulnerability you see. Your peers will tell you how
   interesting they think this vulnerability is, and you'll add that to the
   upvotes score on your branch.
-upvotes: 
+upvotes:
```
Comment (axt1384) [inline cves/CVE-2017-5038.yml]: I'd give this CVE a 8

File: cves/CVE-2010-3415.yml [hunk 4]
```diff
@@ -218,7 +219,7 @@ mistakes:
     Use those questions to inspire your answer. Don't feel obligated to answer
     every one. Write a thoughtful entry here that those ing the software
     engineering industry would find interesting.
-  answer: "The main reason for this vulnerability was a design mistake. The the bridge
-    map\nwas stored in the wrong location which resulted in stale values. When Chrome
-    tried to access\nthose stale values it resulted in a crash. This could've been
-    prevented by doing a code\nreview and performing a detailed design review. \n"
+  answer: |
+    Design mistakes lead to this vulnerability. The problem was storing an object
+    in an inccorect place. When they were to be accessed they had old values. This
```
[the remark this answers is not in the document]
Comment (VigorousJunk) [reply cves/CVE-2010-3415.yml]: Reworded.

File: cves/CVE-2017-5038.yml (hunk 1 above)
Comment (VigorousJunk) [reply cves/CVE-2017-5038.yml]: Done.

File: cves/CVE-2010-3415.yml (hunk 2 above)
Comment (cjr8920) [inline cves/CVE-2010-3415.yml]: I give this 3 upvotes as it's interesting to see how error messages can do to harm a system

File: cves/CVE-2010-3415.yml (hunk 4 above)
Comment (cjr8920) [inline cves/CVE-2010-3415.yml]: This seems like it could be rewritten as the wording flows a bit oddly.

File: cves/CVE-2017-5038.yml [hunk 5]
```diff
@@ -32,42 +32,52 @@ description_instructions: |
   that outsiders to Chromium would not understand. Technology like "regular
   expressions" is fine, and security phrases like "invalid write" are fine to
   keep too.
-description: 'Use after free in GuestView.
+description: |
+  The Guestview and other related javascript objects were able to inherit from
```
Comment (cjr8920) [inline cves/CVE-2017-5038.yml]: The description could be a little too intricate for someone to understand. Consider rewriting to make it more understandable.

File: cves/CVE-2017-5038.yml (hunk 3 above)
Comment (cjr8920) [inline cves/CVE-2017-5038.yml]: I give this CVE a 2

File: cves/CVE-2017-5038.yml [hunk 6]
```diff
@@ -123,10 +139,15 @@ interesting_commits:
     If there are no interesting commits, demonstrate that you completed this section by explaining what happened between the VCCs and the fix.
   answer:
   commits:
-  - commit: 
-    note: 
-  - commit: 
-    note: 
+  - commit: b0459422d0c363d4c7208916f00eee74d0d503ea
+    note: |
+      Uses function wrappers to avoid memory leaks. Interesting that memory
```
Comment (cjr8920) [inline cves/CVE-2017-5038.yml]: "Interesting..." isn't a complete sentence.

File: cves/CVE-2017-5038.yml [hunk 7]
```diff
@@ -135,12 +156,16 @@ major_events:
 
     The event doesn't need to be directly related to this vulnerability, rather,
     we want to capture what the development team was dealing with at the time.
-  answer: 
+  answer: |
+    No major events but something I found intersting over the course of the
```
Comment (cjr8920) [inline cves/CVE-2017-5038.yml]: This section has a few misspellings and missing punctuation.

File: cves/CVE-2010-3415.yml (hunk 2 above)
Comment (axt1384) [inline cves/CVE-2010-3415.yml]: I'll change my upvote to a 2, since the upvotes are spread across all CVEs

File: cves/CVE-2017-5038.yml (hunk 3 above)
Comment (axt1384) [inline cves/CVE-2017-5038.yml]: I'll change my upvote to a 2, since the upvotes are spread across all CVEs

File: cves/CVE-2010-3415.yml (hunk 4 above)
Comment (VigorousJunk) [reply cves/CVE-2010-3415.yml]: Agreed. I have restructured the sentences

File: cves/CVE-2017-5038.yml (hunk 5 above)
Comment (VigorousJunk) [reply cves/CVE-2017-5038.yml]: I removed chromium specific wording which should make it more easier.

**Comment by exl5325** [comment]
1 upvote

File: cves/CVE-2010-3415.yml (hunk 2 above)
Comment (rburchlau001) [reply cves/CVE-2010-3415.yml]: I will give this 4 upvotes

File: cves/CVE-2010-3415.yml [hunk 8]
```diff
@@ -32,12 +32,10 @@ description_instructions: |
   that outsiders to Chromium would not understand. Technology like "regular
   expressions" is fine, and security phrases like "invalid write" are fine to
   keep too.
-description: "The Geolocation module wasn't correctly implemented. This allowed remote
-  attackers to cause denial of service through memory corruption. \nA fix from a previous
-  bug didn't fix the real issue. A developer was able to come up with the same bug
-  \ but in a different location. The bridges in the bridgemap\nare never removed.
-  The bridge map is currently being stored outside the webcore(a component of webit),
-  which \nmeans we are storing stale pointers. \n"
+description: |
```
Comment (rburchlau001) [inline cves/CVE-2010-3415.yml]: I really like this description, informative and concise

File: cves/CVE-2017-5038.yml [hunk 9]
```diff
@@ -135,12 +157,16 @@ major_events:
 
     The event doesn't need to be directly related to this vulnerability, rather,
     we want to capture what the development team was dealing with at the time.
-  answer: 
+  answer: |
+    No major events, but something I found intersting over the course of the development
+    was that everyone was essentially working on refactoring one persons code. User PaulMeyer
+    seemed to have commited the base and made lots of refactors to his own code
+    along with other members of the team.
```
Comment (rburchlau001) [inline cves/CVE-2017-5038.yml]: Was there a disparity between PaulMeyer changes and the number of others that made them? What was the ratio?

**Pull request merged**